src/NovaS3MultipartUpload.php

```diff
@@ -506,6 +507,17 @@ public function fileManagerSelectionType($string)
         return $this->withMeta(['fileManagerSelectionType' => $string]);
     }
 
+    /**
+     * The content disposition for the files.
+     *
+     * @param  string  $type  (‘view’, ‘download’, ‘both’)
```

Comment:
Better use array e.g. ['inline'] or ['attachement'] or ['inline', 'attachement'] then pass the value directly to `withMeta`